content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -0,0 +1,139 @@
+
+Once again, click :guilabel:`Send by Email`. Next, create a second alternative by
+clicking the :guilabel:`Alternatives` tab, and click :guilabel:`Create Alternative`. When the popup
+appears again, choose a different vendor from the drop-down menu next to :guilabel:`Vendor`, and
+this time, *uncheck* the :guilabel:`Copy Products` checkbox. Click :guilabel:`Create Alternative`.
+
+.. tip::
+    If one or two alternative quotations need to be removed from the :guilabel:`Alternatives` tab,
+    they can be individually removed by clicking on the black "X" icon at the end of their row.
+
+This will create a new purchase order, but since the product quantities of the original
+:abbr:`PO (purchase order)` were not copied over, new products will need to be added to the product
+lines. Add the products, then click :guilabel:`Send by Email`.
+
+From this newest purchase order, click the :guilabel:`Alternatives` tab. From here, all three
+purchase orders can be seen, inluding which vendors they were sent to, the expected arrival of the
+products if confirmed, and the status of the order. Before comparing any product lines, click
+:guilabel:`New` at the top of the page. This will create a brand new quotation. Fill out the
+information on this new quotation, and click :guilabel:`Send by Email`. Then, click the
+:guilabel:`Alternatives` tab once more. On this new :abbr:`PO (purchase order)`, there are no other
+orders linked. To link this order with the alternatives, click :guilabel:`Link to Existing RfQ`.
+
+.. image:: calls_for_tenders/calls-for-tenders-link-existing-rfq.png
+   :align: center
+   :alt: Popup to link new quotation to existing RFQs.
+
+This will cause a popup to appear. From the popup, select the three purchase orders created
+previously, and click :guilabel:`Select`. All of these orders have now been copied to this
+:abbr:`PO (purchase order)` under the :guilabel:`Alternatives` tab.
+
+Compare product lines
+---------------------
+
+Now that there are multiple RFQs linked as alternatives, they can be compared to determine which
+vendors offer the best deal. Under the :guilabel:`Alternatives` tab in the newest purchase order,
+click :guilabel:`Compare Product Lines`.
+
+.. image:: calls_for_tenders/calls-for-tenders-compare-product-lines.png
+   :align: center
+   :alt: Compare Product Lines page for alternative RFQs.
+
+From the *Compare Order Lines* page, each product included in any of the purchase orders will be
+shown in a drop-down, with all of the orders they were included in listed below them. In the
+different columns, the quantities, unit price, and total price of the products can be compared. At
+the end of each row, products can be selected by clicking :guilabel:`Choose`. Once all the desired
+products have been chosen, click :guilabel:`Requests for Quotation` in the breadcrumbs at the top of
+the page to navigate back to an overview of all RFQs.
```

Comment:
different columns, the quantities, unit price, and total price of the products can be compared. 

At the end of each row, products can be selected by clicking :guilabel:`Choose`. Once all the desired
products have been chosen, click :guilabel:`Requests for Quotation` (in the breadcrumbs, at the top of
the page) to navigate back to an overview of all RFQs.